pages/deployments/deploying_to_kubernetes.md.erb

```diff
@@ -0,0 +1,222 @@
+# Deploying to Kubernetes
+
+<%= estimated_time "30 minutes" %>
+
+<%= toc %>
+
+This tutorial shows how to deploy to Kubernetes using separate test and deploy
+[pipelines][]. Separating pipelines like this is best practice. The test
+pipeline runs tests and push a Docker image to a registry. The deploy pipelines
+uses the `DOCKER_IMAGE` enviornment variable to create a [Kubernetes
+deployment][k8s_deployment] via `kubectl`. Then, you'll see how to link them
+together to automate deploys from master.
+
+First up, you need to add a step to your existing test pipline that pushes a
+Docker image. Also check your agents have `kubectl` access to your target
+cluster. Refer to the notes at the end of tutorial for tips on setting this
+up.
+
+<%= image "final_test_pipeline.png", width: 736, height: 199, alt: 'Final Test Pipeline' %>
+
+Let's go.
+
+## Create the Deploy Pipeline
+
+This section covers creating a new Buildkite pipeline that loads steps from
+`.buildkite/pipeline.deploy.yml`. Deploy pipelines are like any other pipeline
+except they're triggred by events other than code pushes. We'll use a [trigger
+steps](https://buildkite.com/docs/pipelines/trigger-step) later on to connect
+the test and deploy pipelines.
+
+Create a [new pipeline](https://buildkite.com/new) for your repo. Use
+`buildkite-agent pipeline upload .buildkite/pipeline.deploy.yml` for the first
+step as shown in the image.
```

Comment:
Best not to rely on people being able to see images and have a more fleshed out description of what they're doing so that the image is just support. 
Rather than:

> Use `buildkite-agent pipeline upload .buildkite/pipeline.deploy.yml` for the firststep as shown in the image

What do you think about something like:

> The first step will be a pipeline upload using our new deploy pipeline.yml file. Enter `buildkite-agent pipeline upload .buildkite/pipeline.deploy.yml` in the `commands to run` field.